CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ The releases of Vieb aim to follow [semantic versioning](https://semver.org).
 - Mouse back/forward buttons being ignored when the mouse setting is enabled
 - Modifiers being ignored when pressing certain named keys such as "Space"
 - Border of split pages moving the page slightly when switching (border is now always there but in gray)
+- Updating the adblocker files when installed on a read-only file system
```

Comment:
I am not really familiar with NixOS, so correct me if I'm wrong, but shouldn't it be `when installed FROM a read-only file system`? To me it seems kind of impossible to install TO a read-only file sytem, and the fixes in this PR make more sense if it's FROM a read-only file system.

Please do tell me which of these is true, so I can better review this PR. I'll also try to read a bit about NixOS myself.

Reply:
Ok, so this wasn't clear enough.  I've rephrased it to "when *Vieb is* installed on a read-only file system".
The issue happens when the vieb executable (and the bundled blocklists) are read-only.

Reply:
Ah, now I understand it a bit more. Thanks for the clarification.